app/modules/pullRequests/actionTypes/taskSaga.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+/* eslint-disable no-multiple-empty-lines, flowtype/require-types-at-top */
+
+import { type TaskSagaAction } from 'types/actions';
+
+
+// Action constants --------------------------------------------------------------------------------
+
+export const FETCH: 'pullRequests/FETCH' = 'pullRequests/FETCH';
+export const SUBMIT: 'pullRequests/SUBMIT' = 'pullRequests/SUBMIT';
```

Comment:
I realize we've sort of had this discussion before about `users#create` vs `users#signup`, but honestly, the more that I think about it the more I'm in favor of just naming all of these `create`, i.e. `topics#create`, `pullRequests#create`, etc. (with `users#signup` as a possible exception since in that case the create process isn't actually complete until the users confirms their account). I mean, what you're doing here is pretty much the same thing for topics as for pull requests: create a new instance of the thing in the backend, fetch it to the frontend, return its id. Imo similar functionality should have similar names. Thoughts?

Reply:
I thought we agreed that task sagas would have a more descriptive story-based name? The argument that `signup` is not fit for administrators creating a third party is not applicable here, either users or administrators could _submit_ a PR.

Reply:
> I thought we agreed that task sagas would have a more descriptive story-based name?

Nope, you decided that on your own :P

> The argument that signup is not fit for administrators creating a third party is not applicable here, either users or administrators could submit a PR.

My point here isn't even about what makes sense from an admin point of view; it's about all of these sagas (`topics#create`, `pullRequests#signup`, etc.) doing essentially the same thing, which is to create the thing in the backend, then fetch the newly created thing and set it in the redux store. Doesn't it make more sense then to also give them the same name?

Reply:
Of course it does, and it's exactly how the API was structured. If you want to have this uniform mapping of CRUD actions onto the API, I'm absolutely fine with that.